### PR TITLE
Fix Login Page (#3742)

### LINKED ISSUE #3742: Logon page strange
### Describe the problem

IMO, this is backward. App info(title box) should be on left because UX for most is immediate look to right for fields. Did a lefty do this?:)
image

### Describe the solution you'd like
Swap

### PR DESCRIPTION
**Pull request by tywrenn**
Fixes #3742 

#### Short description of what this resolves:
Allows customization of the login page to be whatever direction you want it to be.

below added by bradymiller:
Up For Grabs demo for this PR is here:
https://www.open-emr.org/wiki/index.php/Development_Demo#Delta_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by tywrenn** [comment]
@bradymiller Ready for review

**Comment by bradymiller** [comment]
very neat stuff

Up For Grabs demo for this PR is here:
https://www.open-emr.org/wiki/index.php/Development_Demo#Delta_-_Up_For_Grabs_Demo

**Comment by tywrenn** [comment]
@bradymiller A big issue i'm getting regardless of whether the login page is centered is overflow that isn't scrollable. This might be due to how I coded it...i'll investigate further.

File: interface/globals.php [hunk 1]
```diff
@@ -498,7 +498,7 @@ function GetCallingScriptName()
 //
 $top_bg_line = ' bgcolor="#dddddd" ';
 $GLOBALS['style']['BGCOLOR2'] = "#dddddd";
-$logocode = "<img class='img-fluid' src='" . $GLOBALS['OE_SITE_WEBROOT'] . "/images/login_logo.gif' />";
+$logocode = "<img class='img-responsive' src='" . $GLOBALS['OE_SITE_WEBROOT'] . "/images/login_logo.gif' />";
 // optimal size for the tiny logo is height 43 width 86 px
```
Comment (stu01509) [inline interface/globals.php:502]: I just curious, why we change the `img-fluid` to `img-responsive`?

Comment (tywrenn) [reply interface/globals.php:502]: Fluid attempts to get the width of the image to 100% while `img-responsive` tends to not do that

Comment (tywrenn) [reply interface/globals.php:502]: Usually I just go back and forth between the two and see which one works better

**Comment by tywrenn** [comment]
@bradymiller Fixed the overflow issue & ready for review again

**Comment by bradymiller** [comment]
hi @tywrenn , looks good and testing well. bringing into codebase. thanks for the improvements! -brady

**Comment by bradymiller** [comment]
@tywrenn, actually, there's a merge conflict that cropped up from last commit. I am clueless on the scss, so will let you fix it up :)

**Comment by tywrenn** [comment]
Lemme test this first

**Comment by tywrenn** [comment]
@bradymiller Just tested it and ready to test/merge.

**Comment by bradymiller** [comment]
hi @tywrenn , Bringing this in. Thanks for the improvements! -brady

**Pull request merged**